packages/core/test/node/com.spec.ts

```diff
@@ -160,6 +160,25 @@ describe('Communication', () => {
         expect(handleMessageStub).to.have.not.been.called;
     });
 
+    it('serialization of arguments for remote API calls should handle undefined values ', async () => {
```

Comment:
`supports default function parameters in remote API calls`

Reply:
@alisey fixed